apache2/waf_logging/waf_log_util.cc

```diff
@@ -16,6 +16,7 @@ struct detail
     jsoncons::string_view data;
     jsoncons::string_view file;
     jsoncons::string_view line;
+    jsoncons::string_view timestamp;
```

Comment:
This field is unused - please either remove or initialize if it should be used (from the code seems it should not).